CIP-0143/README.md

```diff
@@ -0,0 +1,192 @@
+---
+CIP: 143
+Title: Interoperable Programmable Tokens
+Category: Tokens
+Status: Proposed
```

Comment:
```suggestion
Status: Inactive (incorporated into candidate CIP-0113)
```
As I see it, this is the only 100% honest `Status:` we can assign... though (_note to editors!_) we will have to remember to go back & remove the word `candidate` immediately after #444 is finally merged.